core/src/main/java/nl/nn/adapterframework/pipes/FixedResultPipe.java

```diff
@@ -222,31 +244,48 @@ public void setFileNameSessionKey(String fileNameSessionKey) {
 		setFilenameSessionKey(fileNameSessionKey);
 	}
 
-	@IbisDoc({"name of the session key containing the file name of the file containing the result message", ""})
+	/**
+	 * Name of the session key containing the file name of the file containing the result message.
+	 * Path is relative to the configuration's root directory (file is expected on the classpath).
+	 */
```

Comment:
```suggestion
	/**
	 * Name of the session key containing the file name of the file containing the result message.
	 */
```
Het pad is inderdaad standaard relatief aan de configuratie root directory. Dat is altijd, overal zo. Ik vind het onhandig dat bij elk attribuut te zetten.

Reply:
Verwerkt.